media-libs/mesa/mesa-13.0.5.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=5
 
-EGIT_REPO_URI="git://anongit.freedesktop.org/mesa/mesa"
+EGIT_REPO_URI="https://anongit.freedesktop.org/${PN}/${PN}"
```

Comment:
fatal: repository 'https://anongit.freedesktop.org/mesa/mesa/' not found

Also, please don't put `${PN}` all over the place. It makes it impossible to copy-paste the URL.